src/test/regress/sql/window_functions.sql

```diff
@@ -616,3 +616,21 @@ GROUP BY 1 ORDER BY 1;
 select null = sum(null::int2) over ()
 from public.users_table as ut limit 1;
 
+-- verify that this doesn't crash with DEBUG4
+SET log_min_messages TO DEBUG4;
```

Comment:
maybe also add logging to the query that failed originally

Reply:
Added, thank you!